core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/conditional/SearchResultExpression.kt

```diff
@@ -0,0 +1,68 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.conditional
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.resolvable.expression.UnaliasedExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.toDopeType
+import ch.ergon.dope.validtype.ValidType
+
+class SearchResultExpression<T : ValidType, U : ValidType>(
```

Comment:
1. I would only add the suffix "Expression" to a class name, when it extends `Expression`. Maybe just call it SearchResult.
2. Extend `Resolvable`, that it can override the `toDopeType` function

Reply:
_Resolvable_ basically means it is something which can be converted to a query-string. Even though this class is not an expression it still generates a database-dependent string. I would rather convert this class to a simple data class with two parmas or (as Jan suggested) add the resolvable-interface (in that case please adjust the SetAssignment-class as well)

Reply:
Thanks for the explanation! I will therefore add it to `SetAssignment` as well.